templates/webhook.service.erb

```diff
@@ -3,19 +3,16 @@ Description=R10K Webhook Service
 After=syslog.target network.target
 
 [Service]
-Type=forking
+Type=simple
 EnvironmentFile=-/etc/sysconfig/webhook
 RuntimeDirectory=webhook
 User=<%= @user %>
-PIDFile=/var/run/webhook/webhook.pid
```

Comment:
I think this line was only a problem in CI in docker.  On real systems it would be fine.